pandas/core/internals/managers.py

```diff
@@ -330,18 +330,31 @@ def _verify_integrity(self) -> None:
                 f"tot_items: {tot_items}"
             )
 
-    def reduce(self: T, func) -> T:
+    def reduce(self: T, func, ignore_failures: bool = False) -> Tuple[T, np.ndarray]:
```

Comment:
can you add a doc-string, ideally type func

Reply:
updated+green